bom-weekly/pom.xml

```diff
@@ -400,6 +400,12 @@
         <artifactId>authorize-project</artifactId>
         <version>1.7.1</version>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>aws-credentials</artifactId>
+        <!-- TODO https://github.com/jenkinsci/aws-credentials-plugin/pull/229 -->
```

Comment:
Currently blocked on the merge/release of https://github.com/jenkinsci/aws-credentials-plugin/pull/229